src/app/compare/Compare.m.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 .sorted {
-  color: $orange;
+  color: var(--theme-accent-primary);
```

Comment:
It might be nice to toggle between the primary and secondary accent here based on the sort direction, like we do in [the item sort list](https://github.com/guise/DIM/blob/sass-to-css-variables/src/app/settings/SortOrderEditor.scss#L42-L47).

Reply:
Could you point me to where this `.sorted` element is shown in the app? 

I'd just swapped this variable instance with a find/replace rather than check the context

Reply:
Nice, works for me 🚀